components/tabs/stories/tabs-horizontal-compact.stories.js

```diff
@@ -29,29 +27,54 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
-
-export const WithIcon = Template.bind({});
-WithIcon.args = horizontalWithIconsProps;
-
-export const IconOnly = Template.bind({});
-IconOnly.args = horizontalIconOnlyProps;
-
-export const CompactEmphasized = Template.bind({});
-CompactEmphasized.args = {
-  isEmphasized: true,
-  ...horizontalProps
+const TabsGroup = ({
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; gap: 2rem;">
+			${Template({
+        ...args
+			})}
+			${window.isChromatic() ?
+				Template({
+          ...args,
+          labelWithIcons: true
+				})
+			: null }
+      ${window.isChromatic() ?
+        Template({
+          ...args,
+          iconOnly: true
+        })
+				: null }
+      ${window.isChromatic() ?
+        Template({
+          ...args,
+          isEmphasized: true
+        })
+      : null }
+      ${window.isChromatic() ?
+        Template({
+          ...args,
+          isEmphasized: true,
+          labelWithIcons: true
+        })
+      : null }
+      ${window.isChromatic() ?
+        Template({
+          ...args,
+          isEmphasized: true,
+          iconOnly: true
+        })
+      : null }
```

Comment:
This is just style preference, aka optional, but you could move these into a const and use lit's `when` here. Something like [Combobox](https://github.com/adobe/spectrum-css/blob/main/components/combobox/stories/combobox.stories.js), but that aligns with the new `window.isChromatic()` setup.